src/components/Recipe.js

```diff
@@ -55,11 +55,29 @@ const Instructions = styled.ul`
   word-spacing: 4px;
 `;
 
+function createIngredientList(recipe) {
+  const items = [];
+
+  for (let i = 1; i <= 20; i += 1) {
+    const strIngredient = recipe[`strIngredient${i}`];
+    const strMeasure = recipe[`strMeasure${i}`];
+
+    if (strIngredient || strMeasure) {
```

Comment:
Check if the ingredient or measure is not empty, otherwise, it will render a list of empty items

<img width="241" alt="image" src="https://user-images.githubusercontent.com/5244986/160561271-0a1c3433-6306-4f39-890f-044ca4e35480.png">